packages/@vue/cli-service/lib/commands/build/resolveLibConfig.js

```diff
@@ -66,6 +69,7 @@ module.exports = (api, { entry, name, formats, filename, 'inline-vue': inlineVue
             inject: false,
             filename: 'demo.html',
             libName,
+            isVue3,
```

Comment:
I think inject `vueMajor` is better, else we may need `isVue4` `isVue5` in the future

Reply:
And we can use this version string in the unpkg link.

Reply:
Nice tips.